transport/rif_comms/node.py

```diff
@@ -303,11 +299,38 @@ def log(self):
         return self._log
 
     def __repr__(self):
-        node = f" node:{pex(self._raiden_service.address)}" if self._raiden_service else ""
+        node = f" RIF Comms Transport node:{pex(self.address)}"
         return f"<{self.__class__.__name__}{node} id:{id(self)}>"
 
 
 class LightClientNode(Node):
 
-    def __init__(self, address: Address, config: dict, auth_params: dict):
+    def __init__(self, address: Address, config: dict):
         Node.__init__(self, address, config)
+
+    def _handle_message(self, message: RaidenMessage):
+        """
+        Handle received Raiden message.
```

Comment:
this method is exactly the same as the base one, but, it does not ack messages and calls the raiden service with the is_light_client flag on True